tests/crypto/test_ccm_mode/testcase.yaml

```diff
@@ -0,0 +1,3 @@
+tests:
+-   test:
+        tags: crypto aes ccm
```

Comment:
agree with @pfalcon , this is also not part of what this change is trying to do.